src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNotes.hooks.ts

```diff
@@ -44,57 +39,33 @@ export const useFeedbackNotes = ({ entity: { id, isFlow } }: FeedbackNotesProps)
     const commonParams = { appletId, params: {} };
 
     if (isFlow && submitId) {
-      getFlowNotes({ ...commonParams, submitId, flowId: id });
-
-      return;
+      getNotes({ ...commonParams, submitId, flowId: id });
```

Comment:
let's move the condition logic `if (isFlow && submitId) {...}` as a part of `getNotes` function